openam-authentication/openam-auth-oath/src/main/java/org/forgerock/openam/authentication/modules/oath/OATH.java

```diff
@@ -769,4 +776,16 @@ private void setLoginTime(AMIdentity id, long time)
         }
         return;
     }
+
+    /**
+     * Perform time constant equality check.
+     * Both values should not be null.
+     *
+     * @param str1 first value
+     * @param str2 second vale
+     * @return true if values are equal
+     */
+    private boolean isEqual(String str1, String str2)   {
+         return MessageDigest.isEqual(str1.getBytes(), str2.getBytes());
```

Comment:
Should use getBytes(StandardCharsets.UTF_8) for both calls instead.

Reply:
Think that's those two fixed